src/System.Private.ServiceModel/tests/Scenarios/Infrastructure/SetupValidationTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using Infrastructure.Common;
+using System;
+using Xunit;
+
+public class SetupValidationTests : ConditionalWcfTest
+{
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
+    [OuterLoop]
+    public static void Root_Certificate_Correctly_Installed()
+    {
+        // *** SETUP *** \\
+        InvalidOperationException exception = null;
+
+        // *** EXECUTE *** \\
+        try
+        {
+            ServiceUtilHelper.EnsureRootCertificateInstalled();
+        }
+        catch (InvalidOperationException e)
+        {
+            exception = e;
+        }
+
+        // *** VALIDATE *** \\
+        // Validate rather than allowing an exception to propagate
+        // to be clear the exception was anticipated. 
+        Assert.True(exception == null, exception == null ? String.Empty : exception.ToString());
```

Comment:
Actually we throw null ref if exception == null if I don't check because both arguments are evaluated before the call is made to Assert.True.  I learned the hard way...